docs/topics/signals.rst

```diff
@@ -52,9 +52,58 @@ Deferred signal handlers
 ========================
 
 Some signals support returning :class:`~twisted.internet.defer.Deferred`
-objects from their handlers, see the :ref:`topics-signals-ref` below to know
-which ones.
+objects from their handlers, allowing you to run asynchronous code that
+does not block Scrapy. If a signal handler returns a
+:class:`~twisted.internet.defer.Deferred`, Scrapy waits for that
+:class:`~twisted.internet.defer.Deferred` to fire.
 
+Let's take an example:
+::
+
+    class SignalSpider(scrapy.Spider):
+        name = 'signals'
+        start_urls = ['http://quotes.toscrape.com/page/1/']
+
+        @classmethod
+        def from_crawler(cls, crawler, *args, **kwargs):
+            spider = super(SignalSpider, cls).from_crawler(crawler, *args, **kwargs)
+            crawler.signals.connect(spider.item_scrapped, signal=signals.item_scraped)
+            return spider
+
+        def item_scrapped(self, item):
+            # Send the scrapped item to the server
+            dfd = treq.post(
```

Comment:
I believe common practice is to call it `d` (but I see why you used `dfd`, it is used in related parts of Scrapy).

Reply:
Shall I change it to `d` or keep it `dfd` then?

Reply:
Okay! I will change it to `d`